### ISSUE TITLE
Bios settings not save to config files

### ISSUE THREAD

**Issue by SkyPirate1** [Speedvicio/MedGuiReborn#14]
Whenever I go to the bios tab and set the file to use, it is not being saved.

Example:
default main config has
pce.cdbios syscard3.pce

I use no-intro named roms, and since it is in the same firmware directory I want this to be saved to per console config like this:
pce.cdbios [BIOS] TurboGrafx CD Super System Card (USA) (v3.0).pce

I can select it in the gui, but after saving the configuration using the save button it becomes lost.  Keeping the gui open and reloading the config shows the original default setting.  But however, if I set the firmware and use the X to close the window, it asks if I want to save the config and I select Yes, it saves the new firmware setting to the main mednafen.cfg and not pce.cfg.

So basically firmware is not being saved to per console config using the save button, but saves to mednafen.cfg using the X.  Very odd.

quick edit- I should also mention that per console config box was checked so as to rule out human error on my part :P

**Comment by Speedvicio** [comment]
In the past I had to change the behavior of medgui in saving bios as the dos window allows the passage of a maximum of 8191 characters. 
By saving all the bios paths, this limit is often exceeded also because all the parameters relating to a single module of the mednafen are also saved.
I don't remember exactly when I changed the code but Medgui now only saves the path of the bios relating to the configuration of the loaded module. 
Example: 
If you are opening the pc engine options but try to save the psx bios path, the psx path will not be stored as only the bios pc engine parameter will be passed.

**Comment by Speedvicio** [comment]
> Whenever I go to the bios tab and set the file to use, it is not being saved.
> 
> Example: default main config has pce.cdbios syscard3.pce
> 
> I use no-intro named roms, and since it is in the same firmware directory I want this to be saved to per console config like this: pce.cdbios [BIOS] TurboGrafx CD Super System Card (USA) (v3.0).pce
> 
> I can select it in the gui, but after saving the configuration using the save button it becomes lost. Keeping the gui open and reloading the config shows the original default setting. But however, if I set the firmware and use the X to close the window, it asks if I want to save the config and I select Yes, it saves the new firmware setting to the main mednafen.cfg and not pce.cfg.
> 
> So basically firmware is not being saved to per console config using the save button, but saves to mednafen.cfg using the X. Very odd.
> 
> quick edit- I should also mention that per console config box was checked so as to rule out human error on my part :P

This should add bioses on "per console config"
Can you test?
[MedGuiR.zip](https://github.com/Speedvicio/MedGuiReborn/files/8734406/MedGuiR.zip)

**Comment by Speedvicio** [comment]
Fixed with https://github.com/Speedvicio/MedGuiReborn/commit/812bdcb0a833c09e08152ed6b2b0f9cebc2e31a5 commit.
Thx @SkyPirate1 to report the bug :)
